.omeroci/app-deps

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
```

Comment:
These variables (`-e`, `-u`, etc) can also be set by `docker`. I might look into refactoring that in my follow-up.